rules/no-unknown-modifiers.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+var createAvaRule = require('../create-ava-rule');
+
+var modifiers = {
+	after: true,
+	afterEach: true,
+	before: true,
+	beforeEach: true,
+	cb: true,
+	only: true,
+	serial: true,
+	skip: true,
+	todo: true
+};
```

Comment:
Was not a performance related decision. I found `modifiers[x]` was just a pretty way to know whether x is in there. But yeah, it's more verbose in the list declaration bit, and a tad more error-prone (`{ ..., value: false, ...`). Changed.